pyanaconda/ui/tui/hubs/__init__.py

```diff
@@ -113,6 +114,12 @@ def input(self, args, key):
                     if not spoke.completed and spoke.mandatory:
                         print(_("Please complete all spokes before continuing"))
                         return False
+            # TRANSLATORS: 'h' to help
+            elif key == C_('TUI|Spoke Navigation', 'h'):
+                if self.has_help:
+                    help_path = ihelp.get_help_path(self.helpFile, self.instclass, True)
+                    self.app.switch_screen_modal(HelpScreen(self.app, help_path))
+                    return True
```

Comment:
I know there's just ``False`` above, but I believe there were some constants for ``input()`` method's return values which made more sense than ``True, False``. Could you please check that and change this and the new ``input`` method added below to use the constants?